streams/ht.m3u

```diff
@@ -27,3 +27,9 @@ https://haititivi.com/website/haitinews/index.m3u8
 https://watch.haitilive.net/freehb/rtvs/index.m3u8
 #EXTINF:-1 tvg-id="" status="online",Télévision La Brise (Ch. 28 Les Cayes) [Not 24/7]
 https://watch.haitilive.net/hml/labrisetv/index.m3u8
+#EXTINF:-1 tvg-id="TelePacific.ht" status="online",Tele Pacific
+https://pacific.zekletech.com:2002/live/acw_01/index.m3u8
+#EXTINF:-1 tvg-id="RTVC.ht" status="online",RTVC
+https://stream-02.nyc.dailymotion.com/sec(6TMM0QrMj9Q4IwNZkxGVRBsgoxL1cYN8V6dMGcWDJXlY8zvr-W7MNm6Z7FkGDVos)/dm/3/x88ehx6/d/live-1.m3u8#cell=lnyc
+#EXTINF:-1 tvg-id="IslandTV.ht" status="online",Island TV
+https://playback2.akamaized.net/streams/30299049_9486754_lsibkm72z0jon6tkpkn_1/exp=1668259101~acl=%2f*~data=hdntl~hmac=e81a0edc3a49474c5979ab71a60a873f97f87ce58f0dbe4d9461205f3ff0648d/media/30299049_9486754_lsibkm72z0jon6tkpkn_1@5256000p.m3u8?dw=14400&ts=1668171600
```

Comment:
I suspect this one being a Livestream channel. Use free.fullspeed instead

Reply:
As I've typed, Lane, feel free to edit my mistakes. Thanks for taking the time.

Reply:
Feel free to correct it.